node/runtime/pangolin/src/lib.rs

```diff
@@ -932,6 +931,15 @@ fn migrate() -> Weight {
 		TECHNICAL_COMMITTEE_OLD_PREFIX,
 	);
 
+	migration::remove_storage_prefix(b"RandomnessCollectiveFlip", b"RandomMaterial", b"");
+
+	// Migrate the version to new style.
+	// But this will also update the version.
+	// To bypass the check `if on_chain_storage_version < 4 {`, put this to the last one.
+	frame_support::migrations::migrate_from_pallet_version_to_storage_version::<AllPalletsWithSystem>(
+		&RocksDbWeight::get(),
+	);
```

Comment:
Can you give more explanation about why put this to the last one besides passing the check? In my mind, we should migration the storage version firstly, then do other pallets migration.

Reply:
```
if on_chain_storage_version < 4 { migrate } else { skip }
```

Reply:
Okay.  Another question: Do we have to do something in the Crab or Darwinia about storage_version migration? If so, don't forget to create an issue.

Reply:
> Okay. Another question: Do we have to do something in the Crab or Darwinia about storage_version migration? If so, don't forget to create an issue.

https://github.com/darwinia-network/darwinia/issues/827 This one